docs/samples/product_recommendation_demo.ipynb

```diff
@@ -200,68 +193,38 @@
       "outputs": [],
       "source": [
         "import tempfile\n",
-        "yaml_config = \"\"\"\n",
-        "# Please refer to https://github.com/feathr-ai/feathr/blob/main/feathr_project/feathrcli/data/feathr_user_workspace/feathr_config.yaml for explanations on the meaning of each field.\n",
+        "yaml_config = f\"\"\"\n",
         "api_version: 1\n",
         "project_config:\n",
-        "  project_name: 'feathr_getting_started'\n",
-        "  required_environment_variables:\n",
-        "    - 'REDIS_PASSWORD'\n",
+        "  project_name: '{FEATHR_PROJECT_NAME}'\n",
         "offline_store:\n",
         "# Please set 'enabled' flags as true (false by default) if any of items under the same paths are expected to be visited\n",
         "  adls:\n",
         "    adls_enabled: true\n",
         "  wasb:\n",
         "    wasb_enabled: true\n",
-        "  s3:\n",
-        "    s3_enabled: false\n",
-        "    s3_endpoint: 's3.amazonaws.com'\n",
-        "  jdbc:\n",
-        "    jdbc_enabled: false\n",
-        "    jdbc_database: 'feathrtestdb'\n",
-        "    jdbc_table: 'feathrtesttable'\n",
-        "  snowflake:\n",
-        "    snowflake_enabled: false\n",
-        "    url: \"<replace_with_your_snowflake_account>.snowflakecomputing.com\"\n",
-        "    user: \"<replace_with_your_user>\"\n",
-        "    role: \"<replace_with_your_user_role>\"\n",
         "spark_config:\n",
         "  spark_cluster: 'azure_synapse'\n",
         "  spark_result_output_parts: '1'\n",
         "  azure_synapse:\n",
-        "    dev_url: 'https://feathrazuretest3synapse.dev.azuresynapse.net'\n",
-        "    pool_name: 'spark3'\n",
-        "    workspace_dir: 'abfss://feathrazuretest3fs@feathrazuretest3storage.dfs.core.windows.net/feathr_getting_started'\n",
+        "    dev_url: 'https://{SYNAPSE_WORKSPACE_NAME}.dev.azuresynapse.net'\n",
+        "    pool_name: 'spark31'\n",
+        "    workspace_dir: 'abfss://{ADLS_FS_NAME}@{ADLS_ACCOUNT}.dfs.core.windows.net/feathr_project'\n",
         "    executor_size: 'Small'\n",
         "    executor_num: 1\n",
-        "  databricks:\n",
```

Comment:
Not sure if I understand it correctly. Databricks settings are removed, so this notebook will be a synapse e2e sample, right? If so, do we need to move this sample to a `synapse` subfolder like Databricks sample do?
https://github.com/feathr-ai/feathr/tree/main/docs/samples/databricks

Reply:
Good idea, moved it. Once this PR merges, I will make PRs for other notebooks too. May be we need to add similar notebooks for databricks too.